### ISSUE TITLE
Test ArUco detection on the robot

### ISSUE THREAD

**Issue by vvasco** [robotology/assistive-rehab#189]
Given the results of this #180, we want to test the ArUco detection on the robot to understand up to which distance the detection is reliable. If such distance is long enough (> `1m` between robot-line), the detection could be acceptable for the TUG, considered that:
- the line should be detected reliably just **once**;
- once detected, as the robot moves, it knows where the line is with respect to it and to the skeleton, even if the line is **not visible**.

I will test the detection for both resolutions `320x240` and `640x480` px.

**Comment by vvasco** [comment]
This is the detection we have on the robot when the line is at:
- ~`2m` from the robot, with a resolution of `320x240` px:
<img src="https://user-images.githubusercontent.com/9716288/57227939-d9f5d600-7012-11e9-878c-45b2e4950027.gif" width="400">

- ~`2.5m` from the robot: 
        -  with a resolution of `320x240` px:
            <img src="https://user-images.githubusercontent.com/9716288/57227946-deba8a00-7012-11e9-8433-4e9657752a96.gif" width="400">
        - with a resolution of `640x480` px:
           <img src="https://user-images.githubusercontent.com/9716288/57228785-ea0eb500-7014-11e9-8423-53836e047f6f.gif" width="400">

The pose estimated at ~`2.5m` at a resolution  of `640x480` px is the best one and the most stable. However, the pose estimated at ~`2m` at a resolution of `320x240` px looks good as well and `2m` distance can be sufficient for the TUG test. Furthermore, since we require a single robust detection, we can grab more poses to get a final robust pose. 
To sum up: line at ~`2m` at a resolution of `320x240` px seems the best option.

**Comment by vvasco** [comment]
This is using a board of `6x1` smaller markers:
<img src="https://user-images.githubusercontent.com/9716288/57235114-6fe52d00-7022-11e9-9e1c-4a50f2d8a2e9.gif" width=400>

The pose looks good. So far, this is the best board I could find (I tried smaller markers, but the line has to be too close to the robot), therefore I'll use this one as default.
